tests/endToEnd.py

```diff
@@ -212,13 +218,15 @@ def testEndToEnd(self):
         #        datarel needs imsim_*.
         ver = 'imsim-2010-12-17-1'
         print "Setting up astrometry_net_data", ver
+        # XXX what is actually used from this setup -- a path in the env?
         ok, version, reason = eups.Eups().setup("astrometry_net_data", versionName=ver)
         if not ok:
             raise ValueError("Couldn't set up version '%s' of astrometry_net_data: %s" % (ver, reason))
 
         self.assert_(eups.Eups().isSetup("obs_lsstSim"))
```

Comment:
I'm not sure if that assertion has much value since lsst.obs.lsstSim is imported on line #49. https://github.com/lsst/testing_endtoend/blob/tickets/DM-2636/tests/endToEnd.py#L49